analyzer/src/main/kotlin/managers/utils/NuGetSupport.kt

```diff
@@ -343,7 +343,7 @@ private fun resolveLocalSpec(definitionFile: File): File? =
 private fun getIdentifier(name: String, version: String) =
     Identifier(type = "NuGet", namespace = "", name = name, version = version)
 
-private fun NuGetDependency.getId() = Identifier("NuGet::$name:$version")
+private fun NuGetDependency.getId() = getIdentifier(name, version)
```

Comment:
It really makes no sense to keep this separate from the previous commit. Please squash it.

Reply:
It really makes no sense to discuss this any further. See may other comment. Let's just disagree and get on with more important stuff.